### PR TITLE
PLATFORM-1860 add of tests for  edge router

### PR DESCRIPTION
**Pull request by pguinard-public-com**
Initial add of tests.

### CODE REVIEW

File: edge_router_public_com/requirements-dev.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+autopep8==2.1.*
+pytest-cov==5.0.*
+pylint==3.1.*
```
Comment (nilspublic) [inline edge_router_public_com/requirements-dev.txt:3]: If we're not linting, we don't need this

If/when we are linting, we'll need the `pylint_junit` module too

**Review by nilspublic — APPROVED** [review]
